fedimint-client/src/module/recovery.rs

```diff
@@ -102,3 +102,51 @@ impl IntoDynInstance for NoModuleBackup {
         DynModuleBackup::from_typed(instance_id, self)
     }
 }
+
+/// Progress of the recovery
+///
+/// This includes "magic" value: if `total` is `0` the progress is "not started
+/// yet"/"empty"/"none"
+#[derive(Debug, Copy, Clone, Encodable, Decodable)]
+pub struct RecoveryProgress {
+    pub complete: u32,
+    pub total: u32,
+}
```

Comment:
What's the unit? Does it matter? I guess not, but still good to know. Unless we actively don't commit to any particular meaning and adopt a notion of "work units".

Reply:
units would depend on the module

Reply:
> units would depend on the module

Yeah. I didn't want to handle the unit explicitly as then it gets complicated (translate the units, plural/singular, etc.) so I just let the module pick the numerator and denominator are hopefully it's will be good enough UX.

Can be improved later.